src/mammos_entity/_base.py

```diff
@@ -290,19 +314,19 @@ def __repr__(self) -> str:
         args = [f"ontology_label='{self._ontology_label}'", f"value={self.value!r}"]
         if str(self.unit):
             args.append(f"unit='{self.unit!s}'")
+        if self.description:
+            args.append(f"description='{self.description}'")
```

Comment:
```suggestion
            args.append(f"description={self.description!r}")
```

Please double-check if this produces valid quoting.